patches/content-browser-url_loader_factory_getter.cc.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/content/browser/url_loader_factory_getter.cc b/content/browser/url_loader_factory_getter.cc
+index 4571fbc6765e25b26b3e9007dc335c950a21c9cb..daea1bb69e0af9ed18678a6d1313b8d630aeab1d 100644
+--- a/content/browser/url_loader_factory_getter.cc
++++ b/content/browser/url_loader_factory_getter.cc
+@@ -17,6 +17,7 @@
+ #include "services/network/public/cpp/features.h"
+ #include "services/network/public/cpp/shared_url_loader_factory.h"
+ #include "services/network/public/mojom/network_service.mojom.h"
++#include "brave/services/network/public/cpp/system_request_handler.h"
+ 
+ namespace content {
+ 
+@@ -143,7 +144,7 @@ class ReconnectableURLLoaderFactoryForIOThread::URLLoaderFactoryForIOThread
+       override {
+     DCHECK(BrowserThread::CurrentlyOn(BrowserThread::IO));
+     factory_getter_->GetURLLoaderFactory()->CreateLoaderAndStart(
+-        std::move(receiver), request_id, options, url_request,
```

Comment:
I think this is the wrong place and it should happen in `ReconnectableURLLoaderFactory::CreateLoaderAndStart`, at least in the current code

Reply:
@deeppandya verified that moving this works correctly and adds the handlers to `GetURLLoaderFactoryForBrowserProcess` and other places that use `ReconnectableURLLoaderFactory`

Reply:
actually why do we  need this at all? This only applies to the profile url loader factory which already goes through BraveRequestHandler, right?

Reply:
I think I see what's going on here now so going to clean this patch up, but this PR needs tests

Reply:
https://github.com/brave/brave-browser/issues/48511